fili-core/src/main/java/com/yahoo/bard/webservice/table/availability/ConcreteAvailability.java

```diff
@@ -106,7 +104,7 @@ public SimplifiedIntervalList getAvailableIntervals(DataSourceConstraint constra
         return metadataService.getAvailableIntervalsByTable(name);
     }
 
-    protected Set<String> getColumnNames() {
-        return columnNames;
+    protected Set<String> getColumnPhysicalNames() {
```

Comment:
Should we rename this `physicalColumnNames`? `columnPhysicalNames` feels a bit backwards, for some reason.